app/templates/keyboard_shortcuts.html

```diff
@@ -0,0 +1,60 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div id="accordion">
+        <div class="card">
+            <div class="card-header" id="General">
+                <h5 class="mb-0">
+                    <button class="btn btn-link" data-toggle="collapse" data-target="#GeneralShortcuts"
+                            aria-expanded="true" aria-controls="collapseOne">
+                        General Shortcuts - Navigation
+                    </button>
+                </h5>
+            </div>
+            <div class="collapse" id="GeneralShortcuts">
+                <div class="card card-body">
+                    <table class="table table-sm">
+                        <thead>
+                        <tr>
+                            <th scope="col">Action</th>
+                            <th scope="col">Shortcut</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        <tr>
+                            <td>Open Home</td>
+                            <td>Press Alt+C+H</td>
+                        </tr>
+                        <tr>
+                            <td>Open Profile</td>
+                            <td>Press Alt+C+P</td>
+                        </tr>
+                        <tr>
+                            <td>Open Agenda</td>
+                            <td>Press Alt+C+A</td>
+                        </tr>
+                        <tr>
+                            <td>Open Invitations</td>
+                            <td>Press Alt+C+I</td>
+                        </tr>
+                        <tr>
+                            <td>Open Search</td>
+                            <td>Press Alt+C+S</td>
+                        </tr>
+                        <tr>
+                            <td>Open All Shortcuts</td>
+                            <td>Press Ctrl+.</td>
+                        </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
```

Comment:
We try not to use `jQuery` if it is possible

Reply:
Thanks, I changed